galasa-simbank-tests/dev.galasa.simbank.manager/src/main/java/dev/galasa/simbank/manager/internal/SimBankManagerImpl.java

```diff
@@ -197,6 +211,55 @@ public void initialise(@NotNull IFramework framework, @NotNull List<IManager> al
         }
     }
 
+    @Override
+    public void executeGherkin(@NotNull IGherkinExecutable executable, Map<String, Object> testVariables)
```

Comment:
Think we are going to need a better way of registering statements to a manager,   don't think simply registering a manager is sufficient as you have shown it is a pain to have to do the regex twice.
Maybe IStatementOwningManager,  which one method  getManager().   The Manager can then implement to contain all the information it gathered when registering so doesn't have to do it twice.